tracing/providers/opentracing/src/main/java/io/helidon/tracing/providers/opentracing/OpenTracing.java

```diff
@@ -44,7 +59,31 @@ public static io.helidon.tracing.Tracer create(Tracer tracer) {
      * @return Helidon {@link io.helidon.tracing.Span}
      */
     public static Span create(Tracer tracer, io.opentracing.Span span) {
-        return new OpenTracingSpan(tracer, span);
+        return new OpenTracingSpan(tracer, span, SPAN_LISTENERS.get());
+    }
+
+    static void invokeListeners(List<SpanListener> spanListeners, System.Logger logger, Consumer<SpanListener> operation) {
+        List<Throwable> throwables = new ArrayList<>();
```

Comment:
Same as above - this method will be invoked very often, should skip all the below if `spanListeners` is empty (and should not create a new list of throwables)

Reply:
done